### PR TITLE
FLIP 212: Reject references to references

### PR DESCRIPTION
**Pull request by turbolent**
See https://github.com/onflow/cadence/issues/2873

### CODE REVIEW

**Review by SupunS — APPROVED** [review]
+1 for the proposal.

I think trying to support reference to references "correctly" is quite difficult and can leave room for many edge cases, which doesn't seem worth considering their (un)usefulness.

**Comment by bjartek** [comment]
+1

**Comment by bluesign** [comment]
+1 and big +1 for `dynamic evaluation rules are changed to reject the creation of references to references`

File: cadence/20231016-reject-references-to-references.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+status: draft
+flip: 211
+authors: Bastian Mueller (bastian@dapperlabs.com)
+sponsor: Bastian Mueller (bastian@dapperlabs.com)
+updated: 2023-10-16
```
Comment (turbolent) [inline cadence/20231016-reject-references-to-references.md:6]:
```suggestion
status: accepted
flip: 211
authors: Bastian Mueller (bastian@dapperlabs.com)
sponsor: Bastian Mueller (bastian@dapperlabs.com)
updated: 2023-10-24
```

**Pull request merged**